src/sfctl/custom_service.py

```diff
@@ -125,7 +125,7 @@ def service_update_flags( #pylint: disable=too-many-arguments
         target_rep_size=None, instance_count=None, rep_restart_wait=None,
```

Comment:
__Line number irrelvant__

Are these changes for the latest SDK? Can they work with 6.1 clusters? If so could you please update the `src/README.rst` file to reflect that this new feature is added.

Reply:
No this is only for 6.2? When should we change the readme in that case?

Reply:
Yes pls :) There should be a section called "unreleased" or something like that. Can you add it in there? Thanks!

Reply:
done